### ISSUE TITLE
De-noising autoencoder pre-training for EEG understanding

### ISSUE THREAD

**Issue by cwbeitel** [projectclarify/pcml#74]
We are interested in unimodal self-supervised learning methods for cortical sensor data that allow us to learn from the variety of large unlabeled data sets available online. One way to do this is with denoising autoencoders including with a variety of possible types of noise (gaussian, channel dropout, single or multi-channel dropout, etc.).

Related to https://github.com/projectclarify/pcml/issues/36
